platform/microprofile/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" version="1.1"
+  bean-discovery-mode="all">
+  <alternatives>
+     <class>org.trellisldp.api.NoopEventService</class>
```

Comment:
I'm not sure why we need this _and_ the `@Produces` method above?

Reply:
You are correct, this is not needed.